runtime/util/fltmath.c

```diff
@@ -37,6 +37,28 @@
 #define J9_NO_DENORMAL_FLOAT_SUPPORT 1
 #endif
 
+I_32
```

Comment:
these can return bool

Reply:
also they can eb moved to valuetype helpers as an inline function since we will only use them in the ValueType case